doc/userguide/output/eve/eve-json-output.rst

```diff
@@ -110,6 +110,8 @@ The most common way to use this is through 'EVE', which is a firehose approach w
             #md5: [body, subject]
 
         - ssh
+        - modbus:
+            array-as-dict: yes       # log values as an integer indexed dict (ELK compliant)
```

Comment:
if I read the code correctly, this controls the type of a field in the json output. I think having a single field that can have multiple types is not good practice.

Reply:
Explanation of the reason for this option to exist is missing.